docs/api.rst

```diff
@@ -266,7 +295,7 @@ periodic
 
   periodic :: Period -> Stream void
 
-Create an infinite :ref:`Stream` containing events that occur at a specified period. ::
+Create an infinite :ref:`Stream` containing events that occur at a specified :ref:`Period`. ::
```

Comment:
Should we add that the first event occurs at time 0?

Reply:
In fact, it's specified twice just below: once in the ascii diagram, and once in the sentence below the ascii diagram.  Maybe the right thing is simply to move that lower sentence up to this first paragraph?  E.g.:

> Create an infinite :ref:`Stream` containing events that occur at a specified :ref:`Period`. The first event occurs at time 0, and the event values are undefined. ::

Reply:
Good idea.

Reply:
Done in 3e90ef4